packages/askui-nodejs/src/execution/ui-controller-client-interface.ts

```diff
@@ -1,6 +1,17 @@
 import { CredentialArgs } from '../utils/http/credentials';
 import { AnnotationLevel } from './annotation-level';
 
+/**
+ * Provide configuration options for the askui UI Control Client
```

Comment:
I would rm `Provide` and uppercase `Configuration` as we are describing something static